### DISCUSSION TITLE
What is the best way to select filters from recently made RTK request.

### DISCUSSION THREAD

**Discussion by AleksandrukTad** [reduxjs/redux-toolkit#2427] [Q&A] [answered]
Hey,
I am looking for a best way to select filters from recently made api request.

The easiest example that comes to my mind is do something like that:
` const filters = useSelector((state: RootState) => Object.values(state.employeeApi.queries)[0].originalArgs.variables.filters)`

or to make it more reusable use createSelector. 

Is there any other *easier* way to do that?

**Comment by phryneas** [comment, marked as answer]
No. I would suggest saving those filters into a custom slice if you need them in multiple places.
